packages/client/README.md

```diff
@@ -100,14 +100,39 @@ if ( ability ) {
 }
 ```
 
-#### `registerAbility( ability: Ability ): void`
+#### `getAbilityCategories(): Promise<AbilityCategory[]>`
+
+Returns all registered ability categories. Categories are used to organize abilities into logical groups.
+
+```javascript
+const categories = await getAbilityCategories();
+console.log( `Found ${ categories.length } categories` );
+
+categories.forEach( ( category ) => {
+  console.log( `${ category.label }: ${ category.description }` );
+} );
+```
+
+#### `getAbilityCategory( slug: string ): Promise<AbilityCategory | null>`
+
+Returns a specific ability category by slug, or null if not found.
+
+```javascript
+const category = await getAbilityCategory( 'data-retrieval' );
+if ( category ) {
+  console.log( `Found category: ${ category.label }` );
+  console.log( `Description: ${ category.description }` );
+}
+```
+
+#### `registerAbility( ability: Ability ): Promise<void>`
```

Comment:
We will need an update in another document the example with missing `await`, too:

https://github.com/WordPress/abilities-api/blob/trunk/docs/7.javascript-client.md#registerability-ability-

A larger question for later is how to avoid redundancy between these two documents. Is this one auto-generated from JSDoc?